tests/shell/main.rs

```diff
@@ -1,18 +1,40 @@
+use assert_cmd::prelude::*;
```

Comment:
The rest of the codebase uses explicit `use` statements without wildcards. IMO it is worth having explicit `use` statements where possible to give people reading the code a better idea of what's in scope. Is it reasonable to turn this `use` statement into explicit ones?

Reply:
Ok updated :)